backbone/src/main/java/org/researchstack/backbone/ui/step/body/TextQuestionBody.java

```diff
@@ -70,7 +71,14 @@ public View getBodyView(int viewType, LayoutInflater inflater, ViewGroup parent)
         // Format EditText from TextAnswerFormat
         TextAnswerFormat format = (TextAnswerFormat) step.getAnswerFormat();
 
-        editText.setSingleLine(!format.isMultipleLines());
+        if(format.isMultipleLines()) {
```

Comment:
What if we change "isMultipleLines" to "lineCount" and default to 1?  I think that would be the best solution moving forward after seeing all the changes you needed to do to editText